src/feature/MovieBrowser/Movies/List/movieListSlice.js

```diff
@@ -5,7 +5,7 @@ const movieListSlice = createSlice({
     initialState: {
         status: "loading",
         movies: [],
-        genres: [],
+        genres: null,
```

Comment:
Great job, Adrian!
I am feeling embarrassed for my code with Peoplelist 😄 
![image](https://user-images.githubusercontent.com/123471023/233846694-0086d317-62a1-4197-8f4e-377eb75f1231.png)